schemas/com.snowplowanalytics.snowplow.badrows/loader_failure_with_unkown_data/jsonschema/1-0-0

```diff
@@ -0,0 +1,45 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Schema for storage loader failures with unknown data, e.g. invalid enriched event",
+  "self": {
+    "vendor": "com.snowplowanalytics.snowplow.badrows",
+    "name": "loader_failure_with_unknown_data",
```

Comment:
Can we come up with a more concise name? At least `enriched_parsing_error` or `loader_parsing_error`, 3 underscores in an entity name is my personal upper bound limit.

Reply:
I think we can go with `enriched_parsing_error`

Reply:
I think we can do better, but if not - let's stick with `enriched_parsing_error`

Reply:
After second thought, I think `enriched_parsing_error` could be confused with `enrichment_failures` which Ben is creating. I think going with `loader_parsing_error` would be better.